sdk/test/IntegrationTests/Tests/SQS.cs

```diff
@@ -265,5 +261,40 @@ private string createQueueTest(string name)
 
             return "fail";
         }
+
+        [TestMethod]
+        [TestCategory("SQS")]
+        public void SQSFIFOTest()
+        {
+            string fifoQueueName = prefix + new Random().Next() + ".fifo";
+
+            var sqsClient = new AmazonSQSClient(RegionEndpoint.USEast2);
```

Comment:
To make sure the client is always disposed, wrap its use in a `using` block:
```
using(var sqsClient = new AmazonSQSClient(RegionEndpoint.USEast2))
{
}
```

This way you don't need to remember the `sqsClient.Dispose()` call and `Dispose` will be called in the event of an exception, like if an `Assert` or the queue creation fails.

The code tests FIFO queues, but for completeness this test should also create a non-FIFO queue and verify that we get expected values for the new properties.